src/pupil_labs/camera/utils.py

```diff
@@ -0,0 +1,68 @@
+import importlib
+import typing as T
+
+import numpy as np
+
+
+class AvailableBackends:
+    @staticmethod
+    def has_mpmath() -> bool:
+        importlib.invalidate_caches()
+        try:
+            importlib.import_module("mpmath")
+            import mpmath
+            import mpmath as _
+
+            from . import backend_mpmath
+            from . import backend_mpmath as _
+
+            return True
+        except ImportError:
+            return False
```

Comment:
I would also like to revisit the `has_x` functions. The tests pass with the following change. Can you confirm that they do for you, too, @romanroibu ?
```suggestion
    def has_mpmath() -> bool:
        try:
            import mpmath

            return True
        except ImportError:
            return False
```
I have applied this change to the other functions as well, and the tests keep passing.

If this is the case for all platforms, I suggest removing the extra import cache invalidations and imports.